### DISCUSSION TITLE
Better exchange of configurations and individual configuration elements: Configuration marketplace

### DISCUSSION THREAD

**Discussion by friebsch** [dainst/idai-field-client#171] [Ideas]
In the future, it must be easy to exchange configurations and individual configuration elements, such as categories and value lists, between projects. Ideally, this also includes a platform where these elements can be previewed and downloaded: A configuration marketplace.

**Comment by lsteinmann** [comment]
@dersmon Suggested the marketplace may be implemented in Field Hub - maybe.
[See also here](https://github.com/dainst/idai-field/milestone/3)
